redpipe/connections.py

```diff
@@ -54,15 +55,31 @@ def get(cls, name: Optional[str] = None) -> redis.client.Pipeline:
         except KeyError:
             raise InvalidPipeline('%s is not configured' % name)
 
+    @classmethod
+    def get_client(cls, name: Optional[str]) -> redis.Redis:
```

Comment:
Should this be a `non-optional str` here?
```suggestion
    def get_client(cls, name: str) -> redis.Redis:
```